### DISCUSSION TITLE
How do I render template now? I cannot get html and css anymore?

### DISCUSSION THREAD

**Discussion by wyxcoder** [GrapesJS/grapesjs#5132] [Q&A]
I used to save template with tags like gjs-html, gjs-css etc. I can then render for frontend with the save html and css.
But now I cannot save with those tags, instead I see tags like assets, styles etc.
So how do I render the template as a html page for end users?

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/discussions/5130#discussioncomment-5967887
